core/templates/base-components/base-content.component.ts

```diff
@@ -66,6 +66,11 @@ export class BaseContentComponent {
       (message: string) => this.loadingMessage = message
     );
     this.keyboardShortcutService.bindNavigationShortcuts();
+
+    // TODO(sll): Use 'touchstart' for mobile.
+    this.windowRef.nativeWindow.document.addEventListener('click', () => {
+      this.sidebarStatusService.onDocumentClick();
+    });
```

Comment:
This code was earlier present in the `Base.ts` file.
Base.ts file is only used in hybrid pages (i.e. pages with both angular and angularjs) but not in angular pages (i.e. pages not using angularjs, about-page in this PR).
Whereas, base-content is used in both. So, I moved the code here, now this functionality works in both hybrid and angular pages without duplication of code.